src/components/locations-list/index.js

```diff
@@ -0,0 +1,3 @@
+import LocationsList from './component';
```

Comment:
I like more:
`export { default } from './component';`
It is a matter of personal preference